extrakto.py

```diff
@@ -111,7 +117,12 @@ def filter(self, text):
             if self.rstrip:
                 item = item.rstrip(self.rstrip)
 
-            if len(item) >= self.extrakto.min_length:
+            # prefer global min_length, fallback to filter specific
+            if len(item) >= (
+                self.extrakto.min_length
+                if self.extrakto.min_length is not None
```

Comment:
This is evaluated every time the function is called. I think it's better to handle this in ~line 68 when creating the filter.

Reply:
Addressed in 6f9341a